revealer_bot/built_in_messages.py

```diff
@@ -1,22 +1,37 @@
 simple_reply_defaults = [
     ("vegetables", "I like vegetables"),
-    ("nft", "Once 10 ETH have been contributed the album will be released and anyone can listen to the album."),
+    (
+        "nft",
+        "Once 10 ETH have been contributed the album will be released and anyone can listen to the album.",
```

Comment:
Uggh, this is ugly - some lines were too long and some weren't, so each line is formatted differently :/